sw/device/silicon_creator/lib/drivers/otbn.c

```diff
@@ -194,10 +196,7 @@ rom_error_t otbn_set_ctrl_software_errs_fatal(bool enable) {
     new_ctrl = 0;
   }
 
-  abs_mmio_write32(kBase + OTBN_CTRL_REG_OFFSET, new_ctrl);
-  if (abs_mmio_read32(kBase + OTBN_CTRL_REG_OFFSET) != new_ctrl) {
-    return kErrorOtbnUnavailable;
-  }
+  sec_mmio_write32(kBase + OTBN_CTRL_REG_OFFSET, new_ctrl);
```

Comment:
We should call `SEC_MMIO_WRITE_INCREMENT()` to increment the expected write count. If you search for its usages, we typically define an enum for the increment value and call this macro in the caller.